docs/hub/post_install.md

```diff
@@ -27,145 +26,13 @@ lastupdated: "2021-02-20"
-   ibm-edge-agbot-5546dfd7f4-4prgr                1/1     Running     0          81m
-   ibm-edge-agbot-5546dfd7f4-sck6h                1/1     Running     0          81m
-   ibm-edge-agbotdb-keeper-0                      1/1     Running     0          88m
-   ibm-edge-agbotdb-keeper-1                      1/1     Running     0          87m
-   ibm-edge-agbotdb-keeper-2                      1/1     Running     0          86m
-   ibm-edge-agbotdb-proxy-7447f6658f-7wvdh        1/1     Running     0          88m
-   ibm-edge-agbotdb-proxy-7447f6658f-8r56d        1/1     Running     0          88m
-   ibm-edge-agbotdb-proxy-7447f6658f-g4hls        1/1     Running     0          88m
-   ibm-edge-agbotdb-sentinel-5766f666f4-5qm9x     1/1     Running     0          88m
-   ibm-edge-agbotdb-sentinel-5766f666f4-5whgr     1/1     Running     0          88m
-   ibm-edge-agbotdb-sentinel-5766f666f4-9xjpr     1/1     Running     0          88m
-   ibm-edge-css-5c59c9d6b6-kqfnn                  1/1     Running     0          81m
-   ibm-edge-css-5c59c9d6b6-sp84w                  1/1     Running     0          81m
-   ibm-edge-css-5c59c9d6b6-wf84s                  1/1     Running     0          81m
-   ibm-edge-cssdb-server-0                        1/1     Running     0          88m
-   ibm-edge-exchange-b6647db8d-k97r8              1/1     Running     0          81m
-   ibm-edge-exchange-b6647db8d-kkcvs              1/1     Running     0          81m
-   ibm-edge-exchange-b6647db8d-q5ttc              1/1     Running     0          81m
-   ibm-edge-exchangedb-keeper-0                   1/1     Running     1          88m
-   ibm-edge-exchangedb-keeper-1                   1/1     Running     0          85m
-   ibm-edge-exchangedb-keeper-2                   1/1     Running     0          84m
-   ibm-edge-exchangedb-proxy-6bbd5b485-cx2v8      1/1     Running     0          88m
-   ibm-edge-exchangedb-proxy-6bbd5b485-hs27d      1/1     Running     0          88m
-   ibm-edge-exchangedb-proxy-6bbd5b485-htldr      1/1     Running     0          88m
-   ibm-edge-exchangedb-sentinel-6d685bf96-hz59z   1/1     Running     1          88m
-   ibm-edge-exchangedb-sentinel-6d685bf96-m4bdh   1/1     Running     0          88m
-   ibm-edge-exchangedb-sentinel-6d685bf96-mxv2b   1/1     Running     1          88m
-   ibm-edge-sdo-0                                 1/1     Running     0          81m
-   ibm-edge-ui-545d694f6c-4rnrf                   1/1     Running     0          81m
-   ibm-edge-ui-545d694f6c-97ptz                   1/1     Running     0          81m
-   ibm-edge-ui-545d694f6c-f7bf6                   1/1     Running     0          81m
-   ```
-   {: codeblock}
-
-   Notes:
-   * For more information about any pods in the **Pending** state due to resource or scheduling issues, see the [cluster sizing](cluster_sizing.md) page. This includes information about how to reduce scheduling costs of components.
```

Comment:
For more information about any pods in the **Pending** state due to resource or scheduling issues, see [cluster sizing](cluster_sizing.md).